### DISCUSSION TITLE
Miniconda 3 Installation

### DISCUSSION THREAD

**Discussion by Yoseph-G** [giswqs/geog-312#16] [Q&A] [answered]
Hello? I am using the Macbook Pro M3. I can see from the terminal that the miniconda was installed properly. However, I cannot see the launching icon on my application list. How can I find it? or what did I miss while installing the software? Thank you!

**Comment by giswqs** [comment, marked as answer]
For MacOS, conda can only be accessed through the terminal. There is no GUI for it.
